controllers/complianceeventsapi/migrations/000001_compliance_history_initial_tables.up.sql

```diff
@@ -27,7 +27,8 @@ CREATE TABLE IF NOT EXISTS policies(
    parent_policy_id INT,
    spec TEXT,
    -- SHA1 hash
-   spec_hash CHAR[40],
+   spec_hash CHAR(40),
+   -- should spec_hash be NOT NULL ?
```

Comment:
@mprahl I was wondering if all fields which are in the UNIQUE clause/thing below should be required...

Reply:
@JustinKuli it's a good question but I think my intent was to allow importing the existing policy history, which we don't have the historical spec values for.

If we decide against doing that, then I think it'd be better as a required field.

Reply:
@JustinKuli let me clarify, I meant direct insertions in the database from the script instead of through the API.

Now that I think of it, let's go ahead and make it required and we can put in a dummy value for when it's historically imported.